src/robusta/integrations/slack/sender.py

```diff
@@ -341,6 +370,93 @@ def __create_links(self, finding: Finding):
 
         return LinksBlock(links=links)
 
+    def __send_tool_usage(self, parent_thread: str, slack_channel: str, tool_calls: List[ToolCallResult]) -> None:
+        if not tool_calls:
+            return
+
+        text = "*AI used info from alert and the following tools:*"
+        for tool in tool_calls:
+            file_response = self.slack_client.files_upload_v2(content=tool.result, title=f"{tool.description}")
+            permalink = file_response["file"]["permalink"]
+            text += f"\n• `<{permalink}|{tool.description}>`"
+
+        self.slack_client.chat_postMessage(
+            channel=slack_channel,
+            thread_ts=parent_thread,
+            text=text,
+            blocks=[
+                {
+                    "type": "section",
+                    "text": {"type": "mrkdwn", "text": text},
+                }
+            ],
+        )
+
+    def send_holmes_analysis(
```

Comment:
I am not in love with this flow but I get it.

Reply:
yeah, it's a different way of handling